src/libretro_core.cpp

```diff
@@ -217,27 +208,17 @@ void retro_get_system_av_info(retro_system_av_info* info) {
 	info->timing.sample_rate = 32768;
 }
 
-void retro_set_environment(retro_environment_t cb) {
-	envCallbacks = cb;
-}
+void retro_set_environment(retro_environment_t cb) { envCallbacks = cb; }
```

Comment:
Cut down on ugly newlines if formatting them like this

Reply:
whoops, clang-format did that